logic/src/commonMain/kotlin/com/wire/kalium/logic/featureFlags/KaliumConfigs.kt

```diff
@@ -0,0 +1,44 @@
+package com.wire.kalium.logic.featureFlags
+
+data class KaliumConfigs(
+    val isChangeEmailEnabled: Boolean = false,
+    val isMarketingCommunicationEnabled: Boolean = false,
+    val isSSoEnabled: Boolean = false,
+    val isAccountCreationEnabled: Boolean = false,
+    val blockOnJailbreakOrRoot: Boolean = false,
+    val blockOnPasswordPolicy: Boolean = false,
+    val forceAppLock: Boolean = false,
+    val forceConstantBitrateCalls: Boolean = false,
+    val forceHideScreenContent: Boolean = false,
+    val forcePrivateKeyboard: Boolean = false,
+    val keepWebSocketOn: Boolean = false,
+    val wipeOnCookieInvalid: Boolean = false,
+    val submitCrashReports: Boolean = false,
+    val webLinkPreview: Boolean = false,
+    val developerFeaturesEnabled: Boolean = false,
+    val isLoggingEnabled: Boolean = false,
+    val isSafeLoggingEnabled: Boolean = false,
+    val enableBlacklist: Boolean = false,
+    val fileRestrictionEnabled: Boolean = false,
+    val countlyAppKey: String = "",
+    val countlyServerUrl: String = "",
+    val customUrlScheme: String = "",
+    val fileRestrictionList: String = "",
+    val httpProxyPort: String = "",
+    val httpProxyUrl: String = "",
+    val websiteUrl: String = "",
+    val webSocketUrl: String = "",
+    val certificate: String = "",
+    val domain: String = "",
+    val supportEmail: String = "",
+    val teamsUrl: String = "",
+    val backendUrl: String = "",
+    val blacklistHost: String = "",
+    val accountUrl: String = "",
```

Comment:
I wonder about these... @MohamadJaara  might help.

I think we configure Kalium to allow or not multiple backends.
But having these Urls here is redundant. As we already pass them when needing to authenticate, etc.

Reply:
we can delete the PROD/STAGING companion object in `ServerConfig` and take whatever is in here as the DEFAULT

Reply:
and we need to add a new  feature flag for en/disable mutable accounts in different BEs (and acutely disable it)

Reply:
> we can delete the PROD/STAGING companion object in `ServerConfig` and take whatever is in here as the DEFAULT

I'd leave them there, but take whatever is defined here as default instead.

Leaving them there makes it easier to bootstrap Kalium into a new client/project.

Reply:
Also, are we meant to initialise the running environment on `WireActivityViewModel.serverConfig` with these values instead?